Content.Server/Cloning/Components/CloningConsoleComponent.cs

```diff
@@ -0,0 +1,17 @@
+using Content.Server.UserInterface;
+using Robust.Server.GameObjects;
+using Content.Shared.Cloning.CloningConsole;
+
+namespace Content.Server.Cloning.CloningConsole
+{
+    [RegisterComponent]
+    public sealed class CloningConsoleComponent : Component
+    {
+        [ViewVariables]
+        public BoundUserInterface? UserInterface => Owner.GetUIOrNull(CloningConsoleUiKey.Key);
+        [ViewVariables]
```

Comment:
github was being shit but pretty sure these would need to be datafields to have machine linking save them.